<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>533</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Шифрование методами перестановки и гаммирования</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Зашифровать</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Расшифровать</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Побитовая перестановка</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Посимвольная перестановка</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Гаммир. с ПСЧ</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Гаммир. с ключом</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Побитовая перестановка</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Посимвольная перестановка</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Гаммир. с ПСЧ</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Гаммир. с ключом</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
